meteor-addons.js

```diff
@@ -1,13 +1,15 @@
 // Meteor specific addons to the bootbox.js object
 bootbox.templateDialog = function(options) {
-  var div = document.createElement("div");
-  UI.insert(UI.renderWithData(options.template, options.data), div);
-  options.message = div;
+
+  var view = Blaze.renderWithData(options.template, options.data);
+
+  options.message = Blaze.toHTML(view);
```

Comment:
Great, give me a chance to test it and I'll pull that in!

On Wed, Jan 14, 2015 at 10:08 AM, boustanihani <notifications@github.com>
wrote:

> In meteor-addons.js
> <https://github.com/cwohlman/meteor-bootboxjs/pull/2#discussion-diff-22946723>
> :
>
> > @@ -1,13 +1,15 @@
> >  // Meteor specific addons to the bootbox.js object
> >  bootbox.templateDialog = function(options) {
> > -  var div = document.createElement("div");
> > -  UI.insert(UI.renderWithData(options.template, options.data), div);
> > -  options.message = div;
> > +
> > +  var view = Blaze.renderWithData(options.template, options.data);
> > +
> > +  options.message = Blaze.toHTML(view);
>
> Thanks for your feedback [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cwohlman/meteor-bootboxjs/pull/2/files#r22946723>.
>